sycl/include/sycl/vector_preview.hpp

```diff
@@ -39,7 +39,7 @@
 #include <sycl/detail/memcpy.hpp>              // for memcpy
 #include <sycl/detail/type_list.hpp>           // for is_contained
 #include <sycl/detail/type_traits.hpp>         // for is_floating_point
-#include <sycl/detail/vector_arith.hpp>        // for vec_arith_common and vec_arith
+#include <sycl/detail/vector_arith.hpp> // for vec_arith_common and vec_arith
```

Comment:
This change is admittedly a little weird, but eventually those comments will disappear so I'm not too concerned.

Reply:
I agree. `clang-format` suggested that change 🤷🏻‍♂️